src/frontend/web_application/src/modules/draftMessage/components/ReplyForm/presenter.jsx

```diff
@@ -174,7 +174,12 @@ class ReplyForm extends Component {
             >
               <Trans id="messages.compose.action.save">Save</Trans>
             </Button>
-            <Button className="m-reply__bottom-action" onClick={this.handleSave} icon="share-alt" responsive="icon-only">
+            <Button
+              className="m-reply__bottom-action"
+              icon="share-alt"
+              responsive="icon-only"
+              disabled
```

Comment:
I prefer commented instead of disabled.

Or notify "not yet implemented" is legit  too.